lib/modules/datasource/conan/index.ts

```diff
@@ -50,6 +55,38 @@ export class ConanDatasource extends Datasource {
     };
   }
 
+  @cache({
+    namespace: `datasource-${datasource}-revisions`,
+    key: ({ registryUrl, packageName }: GetReleasesConfig) =>
+      // TODO: types (#7154)
+      // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+      `${registryUrl}:${packageName}`,
+  })
+  override async getDigest(
+    { registryUrl, packageName, currentDigest }: DigestConfig,
+    newValue?: string
+  ): Promise<string | null> {
+    if (is.undefined(currentDigest) || is.undefined(registryUrl)) {
```

Comment:
```suggestion
    if (is.undefined(newValue) || is.undefined(registryUrl)) {
```

current digest is irrelevant here

Reply:
If there is no `currentDigest`, it means that the user does not want to use them so we should not make the unnecessary API calls

Reply:
renovate will only call getDigest if there is a current digest or the user enabled `pinDigests`.

Reply:
Ok nevermind then